dpnp/dpnp_iface_manipulation.py

```diff
@@ -99,9 +99,9 @@ def asfarray(a, dtype=None, *, device=None, usm_type=None, sycl_queue=None):
     >>> import dpnp as np
     >>> np.asfarray([2, 3])
     array([2.,  3.])
-    >>> np.asfarray([2, 3], dtype='float')
-    array([2.,  3.])
-    >>> np.asfarray([2, 3], dtype='int8')
+    >>> np.asfarray([2, 3], dtype='float16')
+    array([2., 3.], dtype=float16)
+    >>> np.asfarray([2, 3], dtype='int32')
```

Comment:
I believe it would be better to use types dpnp fully supports:
```suggestion
    >>> np.asfarray([2, 3], dtype=dpnp.float32)
    array([2., 3.], dtype=float32)
    >>> np.asfarray([2, 3], dtype=dpnp.int32)
```